config/settings.yml

```diff
@@ -1733,6 +1729,14 @@ vet360:
     enabled: true
     mock: <%= ENV['vet360__veteran_status__mock'] %>
     timeout: 30
+veteran_enrollment_system:
+  associations:
+    api_key: <%= ENV['veteran_enrollment_system__associations__api_key'] %>
```

Comment:
This key needs to be defined in param store.

Reply:
Okay, I will be sure to add it there. Thanks Eric!

Reply:
@ericboehs I added the param here: https://console.amazonaws-us-gov.com/systems-manager/parameters/?region=us-gov-west-1&tab=Table#list_parameter_filters=Name:Contains:veteran_enrollment_system

Reply:
It will also need to be defined for the other environments: dev, staging, sandbox.
![Screenshot 2025-04-16 at 14 31 04@2x](https://github.com/user-attachments/assets/344bd373-7f7c-453c-b2f4-3023e8d6ec8e)

Reply:
Okay. Even if the value is going to be blank/nil, do I still need to add it?

Reply:
As long as the code is okay with it being nil and you're aware it's only going to be available in prod, then you're good.

Reply:
Awesome! Yes, that's the case here. The host for prod is different than for test/dev. The prod host requires the key, the others do not.